sql/example_test.go

```diff
@@ -0,0 +1,59 @@
+package sql_test
+
+import (
+	"context"
+	"database/sql"
+)
+
+/*
+func ExampleResult() {
+	ctx := context.Background()
+	client, err := hazelcast.StartNewClient(ctx)
+	if err != nil {
+		panic(err)
+	}
+	result, err := client.ExecuteSQL(ctx, "SELECT foo FROM bar where zoo=?", 38)
+	if err != nil {
+		panic(err)
+	}
+	defer result.Close(ctx)
+	for result.Next(ctx) {
+		if result.Err != nil {
+			panic(result.Err)
+		}
+		fmt.Println(result.Row.Columns[0])
+		// alternative: fmt.Println(result.Row.ColumnByIndex(0))
+		// alternative: fmt.Println(result.Row.ColumnByName("foo"))
+	}
+}
+
+*/
+
+func ExampleDriver_Open() {
+	db, err := sql.Open("hazelcast", "localhost;Cluster.Unisocket=true")
```

Comment:
It raises an error without port. Maybe we can use default port if not provided?

Reply:
I've changed how default port is handled at https://github.com/hazelcast/hazelcast-go-client/pull/695/commits/c63fe8e2987fe317b8b18296dc3adeb6043e8616